install_deps.sh

```diff
@@ -1,67 +1,97 @@
 #!/bin/bash
 
-set -x -e
+### Extra instructions for docker ###
+# export DEBIAN_FRONTEND=noninteractive TZ=America/Chicago
+# apt update && apt install -y sudo
+
+### Normalize environment ###
+
+set -e
+cd "$(dirname "${0}")"
+
+### Parse args ###
+
+assume_yes=
+while [[ "$#" -gt 0 ]]; do
+    case "${1}" in
+        -y|--yes) assume_yes=true ;;
```

Comment:
This allows one to say `./install_deps.sh --yes` and skip all the prompts. Useful for automated builds :eyes: